bin/oneoff/backfill_data/user_school_infos_backfill

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+
+num_users_updated = 0
+batch_size = 10000
+
+puts "Processing..."
+
+User.where(user_type: "teacher", school_info: !nil).find_in_batches(batch_size: batch_size) do |batch|
```

Comment:
i tested this `.where` query locally and it always returns 0 users. looking at the SQL output:

```
User.where(user_type: "teacher", school_info: !nil)

=> SELECT `users`.* FROM `users` WHERE `users`.`deleted_at` IS NULL AND `users`.`user_type` = 'teacher' AND `users`.`school_info_id` = 1
```

the important part to note is the last part: ```AND `users`.`school_info_id` = 1```. instead of querying for users whose school_info is not nil, it actually interprets that as the integer 1.

i think instead you'll want something like:

```
User.where.not(school_info: nil).where(user_type: "teacher")

=> SELECT COUNT(*) FROM `users` WHERE `users`.`deleted_at` IS NULL AND (`users`.`school_info_id` IS NOT NULL) AND `users`.`user_type` = 'teacher'
```

if you read through the SQL output from that query, it looks correct: grab all active users whose school_info_id is present and whose user_type is "teacher"

Reply:
@madelynkasula Thanks!  Updated code and tested again.